crates/tuono/src/commands/new.rs

```diff
@@ -265,6 +271,22 @@ fn update_cargo_toml_version(folder_path: &Path) -> io::Result<()> {
     Ok(())
 }
 
+fn init_new_git_repo(folder_path: &Path) -> Result<(), io::Error> {
+    let output = Command::new("git").arg("init").arg(folder_path).output()?;
+
+    if output.status.success() {
+        Ok(())
+    } else {
+        Err(io::Error::new(
+            io::ErrorKind::Other,
+            format!(
+                "Git init failed: {}",
+                String::from_utf8_lossy(&output.stderr)
+            ),
+        ))
+    }
+}
```

Comment:
I don't see the point of returning a `Result` for logging a generic trace. Couldn't we manage a more granular logging?

```suggestion
fn init_new_git_repo(folder_path: &Path) {
    if let Ok(output) = Command::new("git").arg("init").arg(folder_path).output() {
       if output.status.is_err() {
          trace!(...)
       }
    } else {
        trace!(...)
    }
}
```